libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -435,15 +434,49 @@ bool AP_Arming::gps_checks(bool report)
             return false;
         }
 
-        // check AHRS and GPS are within 10m of each other
         const Location gps_loc = gps.location();
         Location ahrs_loc;
         if (AP::ahrs().get_position(ahrs_loc)) {
+            // check AHRS and GPS are within 10m of each other horizontally
             const float distance = gps_loc.get_distance(ahrs_loc);
-            if (distance > AP_ARMING_AHRS_GPS_ERROR_MAX) {
+            const uint8_t AHRS_GPS_ERROR_MAX = 10;      // accept up to 10m difference between AHRS and GPS horizontally
```

Comment:
This should just be float, you aren't saving anything by making it uint8_t, other then the compilier having to implicitly convert it for you.

Reply:
Urp.  These were changed from #defines and I only looked at the values we were passing in, not their use :-)

Reply:
Fixed.